### DISCUSSION TITLE
Welcome to underworld3 Discussions!

### DISCUSSION THREAD

**Discussion by lmoresi** [underworldcode/underworld3-archive#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmansour** [comment]
👋

**Comment by julesghub** [comment]
Hello UW3 discussions :wave: :bat: :bathtub:

**Reply by lmoresi** [reply]
😈 

Might be useful as a way of discussing things that are not specifically issues !
